src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2,6 +2,8 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+#nullable enable
```

Comment:
Can you take a look at `InferMethodTypeArguments` (line 5509) and the nullability of its `argumentRefKindsOpt` parameter? We're doing a null test on an ImmutableArray.

Reply:
Sorry, line numbers may be wrong. VS diff isn't smart to keep track in inline-diff view...

Reply:
`ImmutableArray<T>` contains `operator ==` and `operator !=` that treat `null` as equivalent to `IsDefault`.

---
In reply to: [463186065](https://github.com/dotnet/roslyn/pull/45640#discussion_r463186065) [](ancestors = 463186065)